src/_entries/ez-config.md

```diff
@@ -0,0 +1,14 @@
+---
```

Comment:
I haven't tested it, but can you confirm the ErgoDoz-EZ Graphical tool works for the original ErgoDox?